editor/plugins/audio_stream_editor_plugin.cpp

```diff
@@ -109,6 +114,10 @@ void AudioStreamEditor::_play() {
 		_play_button->set_icon(get_theme_icon("MainPlay", "EditorIcons"));
 		set_process(false);
 	} else {
+		if (_current >= _player->get_stream()->get_length()) {
+			_current = 0;
+		}
+		
```

Comment:
There's trailing whitespace here according to https://github.com/godotengine/godot/pull/42122/checks?check_run_id=1124325707